src/aig/gia/giaIf.c

```diff
@@ -2310,6 +2310,8 @@ void Gia_ManTransferPacking( Gia_Man_t * p, Gia_Man_t * pGia )
 }
 void Gia_ManTransferTiming( Gia_Man_t * p, Gia_Man_t * pGia )
 {
+    if ( p == pGia )
```

Comment:
This can happen in a call from `Nf_ManPerformMapping`, and before this fix the PI/PO names would be lost.